SmartDeviceLinkTests/DevAPISpecs/SDLLifecycleProtocolHandlerSpec.m

```diff
@@ -150,6 +150,20 @@ @interface SDLLifecycleProtocolHandler ()
             });
         });
 
+        context(@"no response from the module to the RPC Start Service", ^{
+            beforeEach(^{
+                testHandler.rpcStartServiceTimeoutTimer = nil;
+            });
+
+            it(@"should send a transport disconnected notification when the timer elapses", ^{
+                OCMExpect([mockNotificationDispatcher postNotificationName:[OCMArg isEqual:SDLTransportDidDisconnect] infoObject:[OCMArg isNil]]);
+
+                [testHandler protocolDidOpen:mockProtocol];
+
+                OCMVerifyAllWithDelay(mockNotificationDispatcher, 11.0);
+            });
```

Comment:
This would need to update if you change to `[protocol stop]`

Reply:
Fixed